### PR TITLE
Update gitHub actions

### PR DESCRIPTION
**Pull request by romainfrancois**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -28,7 +28,7 @@ URL: https://dplyr.tidyverse.org,
     https://github.com/tidyverse/dplyr
 BugReports: https://github.com/tidyverse/dplyr/issues
 Depends: 
-    R (>= 3.2.0)
+    R (>= 3.3.0)
```
Comment (hadley) [inline DESCRIPTION:31]: We should leave this at 3.2, unless you have a specific reason to increase the version.

Comment (romainfrancois) [reply DESCRIPTION:31]: The only reason is that 3.2 is no longer tested in actions, but I can revert for sure.

**Comment by romainfrancois** [comment]
Well actually dev vctrs depends on R >= 3.3: https://github.com/r-lib/vctrs/blob/master/DESCRIPTION#L27

**Comment by lionel-** [comment]
@romainfrancois we can change it back to 3.2.

**Pull request merged**